tests/suites/test_suite_pk.function

```diff
@@ -69,6 +69,322 @@ size_t mbedtls_rsa_key_len_func( void *ctx )
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE */
+void valid_parameters( )
+{
+    mbedtls_pk_context pk;
+    size_t len;
+
+    mbedtls_pk_init( &pk );
+
+    TEST_VALID_PARAM( mbedtls_pk_setup( &pk, NULL ) ==
```

Comment:
Please add a test for `mbedtls_pk_free( NULL )`.

Reply:
done